inference-engine/thirdparty/clDNN/src/gpu/ocl_toolkit.h

```diff
@@ -55,6 +55,14 @@ struct gpu_program_state {
         _kernels_cache(context, prog_id) {}
 };
 
+#if (CLDNN_THREADING == CLDNN_THREADING_TBB)
+/// configuration for task_arena
+struct task_arena_config {
+    int max_concurrency;
```

Comment:
Doesn't this parameter duplicate `n_threads` from our engine config?

Reply:
after offline discussion => We are going to move these configs to engine_config.

Reply:
I removed task_arena_config and add core_type to configuration. please review it again thanks.